.github/workflows/publish-artifacts.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: publish artifacts
+# yamllint disable-line rule:truthy
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    branches:
+      # Push events on default branch
+      - devel
+      # Push events to branches matching refs/heads/release-v*
+      - 'release-v*'
+jobs:
+  push:
+    name: Publish artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_IO_USERNAME }}
+          password: ${{ secrets.QUAY_IO_PASSWORD }}
+
+      - name: Set build environment based on Git branch name
+        if: github.ref == 'refs/heads/devel'
+        run: echo "BRANCH_NAME=devel" >> $GITHUB_ENV
```

Comment:
Instead of hardcoding `devel`, maybe use the branch that is set in the environment `${GITHUB_BASE_REF}`?

Reply:
Also, what is `$GITHUB_ENV`? It is not listed on https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
`GITHUB_BASE_REF` is only available on pull_request not on push. GITHUB_ENV is available here https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
thought of doing it but again GITHUB_REF needs some striping and the branch is compared with only `devel` in `deploy.sh` IMO we are good for now. I will check about automating the Image name and the helm chart version from the  branch or the tag name (need more work for that one)